ext/opentelemetry-ext-pymongo/src/opentelemetry/ext/pymongo/__init__.py

```diff
@@ -50,27 +52,51 @@
 COMMAND_ATTRIBUTES = ["filter", "sort", "skip", "limit", "pipeline"]
 
 
-def trace_integration(tracer_provider=None):
-    """Integrate with pymongo to trace it using event listener.
-       https://api.mongodb.com/python/current/api/pymongo/monitoring.html
+class PymongoInstrumentor(BaseInstrumentor):
+    _commandtracer_instance = None
```

Comment:
```suggestion
    _commandtracer_instance: CommandTracer = None
```

Reply:
Manually applied in https://github.com/open-telemetry/opentelemetry-python/pull/612/commits/4b020c778d49f17152408f7a2c8c777c5eb13b21.